lib/linter.js

```diff
@@ -36,6 +37,17 @@ class Linter {
     this.console = options.console || console;
 
     this.loadConfig();
+
+    // we were passed a rule, add the rule being passed in, to the config.
+    // ex:
+    // rule:severity
+    // no-implicit-this:["error", { "allow": ["some-helper"] }]
+    if (options.rule) {
+      const { name, severity, config } = getRuleFromString(options.rule);
+
+      this.config.rules[name] = { severity, config };
+    }
```

Comment:
Can we move this into `loadConfig`? it already uses `this.options` so it has access to all the same info. I'd generally prefer for the contract to be that once `.loadConfig()` is called everything about config loading is done (vs doing somethings in `loadConfig` and other things in this constructor).

Reply:
Okay makes sense, I moved it to loadConfig